php_phongo.c

```diff
@@ -1634,7 +1634,6 @@ static void php_phongo_cursor_iterator_rewind(zend_object_iterator *iter TSRMLS_
 	php_phongo_cursor_iterator *cursor_it = (php_phongo_cursor_iterator *)iter;
 	php_phongo_cursor_t        *cursor = cursor_it->cursor;
 
-	php_phongo_cursor_free_current(cursor);
```

Comment:
Since we can rewind a command cursor multiple times to iterate through the first batch (before we trigger a getmore), wouldn't this invite a memory leak?